### ISSUE TITLE
Can't put maple mini in perpetual bootloader mode

### ISSUE THREAD

**Issue by eagl1** [rogerclarkmelbourne/STM32duino-bootloader#100]
Hi,

Here's my full post in Arduino forum.

[https://forum.arduino.cc/index.php?topic=700036.msg4704325#msg4704325](url)

It was working ok on my PC and now I'm trying to upload a sketch on my laptop but it's not working.
1. Can't do the transfer to perpetual bootloader mode trick. The reset button and the "BUT" are doing the same 6 fast/slow blinks.

2. Can't upload any new sketch.

3. As in my Arduino forum post, hardly get it to be on port COM, after disconnect, it goes to Microchip tools category on windows device manager.

What is the problem ?

**Comment by stevstrong** [comment]
Please use as reference the forum:
https://www.stm32duino.com

**Comment by eagl1** [comment]
> Please use as reference the forum:
> https://www.stm32duino.com
but it's a read only forum, I can't post there !

**Comment by stevstrong** [comment]
You need to register, as on each forum.
Here some readings: https://github.com/rogerclarkmelbourne/Arduino_STM32/wiki

**Comment by eagl1** [comment]
> You need to register, as on each forum.
> Here some readings: https://github.com/rogerclarkmelbourne/Arduino_STM32/wiki

I fired my maple mini this morning. It was stuck in DFU mode and I couldn't upload a sketch to it directly. I had to do rogerclark method of unplug/plug the board after requesting upload a sketch in Arduino IDE and it worked but I tried to flash it or put it in the perpetual mode and I heard a crack and a burnt smell, so I knew I fired it. I have couple new ones :)
